enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,597 @@
+"openshift-machine-config-operator/machine-config-daemon-mhflm"
+"openshift-monitoring/node-exporter-xzwc6"
+"openshift-multus/multus-544fd"
+"openshift-multus/multus-additional-cni-plugins-wkp64"
+"openshift-multus/network-metrics-daemon-cpskx"
+"openshift-network-diagnostics/network-check-target-fq57x"
+"openshift-ovn-kubernetes/ovnkube-node-gpj6p"
+```
+
+### Risks and Mitigations
+
+The main risk in this enhancement is that because we are treating one of the
+master nodes in a 3 node cluster as an arbiter, we are explicitly evicting
+processes that would otherwise be a normal supported upstream configuration such
+as a compact cluster. We run the risk of new components being critical to HA not
+containing the proper tolerations for running on the arbiter node. One of the
+mitigations we can take against that is to make sure we are testing installs and
+updates.
+
+We also need to take stock of pod disruption budgets for platform deployments to
+make sure they can function correctly in this environment. In that same vein we
+also need to check the layered products and validate what we can to make sure
+they can also function in this environment.
+
+A couple of risks we run is customers using an arbiter node with improper disk
+speeds below that recommended for etcd, or a bigger problem being network
+latency. Since etcd is sensitive to latency between members (see:
+[etcd-io/etcd#14501](https://github.com/etcd-io/etcd/issues/14501)), we should
+provide proper guidance so that the arbiter node must meet minimum requirements
+for ETCD to function properly, in disk and network speeds. ([ETCD
+Recommendations](https://docs.openshift.com/container-platform/4.16/scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.html))
+
+### Drawbacks
+
+A few drawbacks we have is that we will be creating a new variant of OpenShift
+that implements a new unique way of doing HA for kubernetes. This does mean an
+increase in the test matrix and all together a different type of tests since the
+addition of an arbiter node will require different validation scenarios for
+failover.
+
+## Open Questions [optional]
+
+1. In the future it might be desired to add another master and convert to a
+   compact cluster, do we want to support changing ControlPlaneTopology field
+   after the fact?
+
+## Test Plan
+
+- We will create a CI lane to validate install and fail over scenarios such as
+  loosing a master or swaping out an arbiter node.
+- Create complimentary lanes for serial/techpreview/no-capabilities.
+- Create a lane for or some method for validating layered products.
+- CI lane for e2e conformance testing, tests that explicitly test 3 node masters
+  will need to be altered to accommodate the different topology.
+- We will add e2e tests to specifically test out the expectations in this type
+  of deployment.
+  - Create tests to validate correct pods are running on the `arbiter`.
```

Comment:
Also need to test that no incorrect pods are running on the arbiter.

Reply:
Sounds good adding to doc.